product_docs/docs/pgd/5/quickstart/quick_start_aws.mdx

```diff
@@ -264,56 +264,59 @@ To leave the SQL client, enter `exit`.
 
 The pgd utility, also known as the PGD CLI, lets you control and manage your EDB Postgres Distributed cluster. It's already installed on the node. 
 
-You can use it to check the cluster's health by running `pgd check-health`:
+You can use it to check the cluster's health by running `pgd cluster show --health`:
 
 ```shell
-pgd check-health
+pgd cluster show --health
 __OUTPUT__
-Check      Status Message
------      ------ -------
-ClockSkew  Ok     All BDR node pairs have clockskew within permissible limit
-Connection Ok     All BDR nodes are accessible
-Raft       Ok     Raft Consensus is working correctly
-Replslots  Ok     All BDR replication slots are working correctly
-Version    Ok     All nodes are running same BDR versions
-enterprisedb@kaboom:~ $
-```
+Connections
```

Comment:
The output needs to be updated. Default output format is tabular. To get the report format output user need to provide `-o markdown` flag